documentation/source/Segmentation.md

```diff
@@ -0,0 +1,152 @@
+    pretrained_weights="cityscapes"         # Drop this line to train from scratch
+)
+```
+Notes:
+- SG includes implementations of 
+[many different architectures](https://github.com/Deci-AI/super-gradients#implemented-model-architectures).
+- Most of these architectures have [pretrained checkpoints](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md) so feel free to experiment!
+
+
+
+### Setup training parameters
+The training parameters includes loss, metrics, learning rates and much more. You can check out the [default training parameters](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/recipes/training_hyperparams/default_train_params.yaml).
+For this task, we will train for 30 epoch, using Binary IoU using the SGD optimizer.
+```py
+from super_gradients.training.metrics.segmentation_metrics import BinaryIOU
+
+train_params = {
+    "max_epochs": 30,
+    "lr_mode": "cosine",
+    "initial_lr": 0.005,
+    "lr_warmup_epochs": 5,
+    "multiply_head_lr": 10,
+    "optimizer": "SGD",
+    "loss": "bce_dice_loss",
+    "ema": True,
+    "zero_weight_decay_on_bias_and_bn": True,
+    "average_best_models": True,
+    "metric_to_watch": "target_IOU",
+    "greater_metric_to_watch_is_better": True,
+    "train_metrics_list": [BinaryIOU()],
+    "valid_metrics_list": [BinaryIOU()],
+    "loss_logging_items_names": ["loss"],
+}
+```
+
+
+### Launch Training
+Now you only need to setup a Trainer and launch the training!
+```py
+from super_gradients import Trainer
+
+trainer = Trainer(experiment_name="segmentation_example", ckpt_root_dir='/path/to/experiment/folder')
+
+trainer.train(model=model, training_params=training_params, train_loader=train_dataloader, valid_loader=valid_dataloader)
+```
+
+
+## Visualize the results
+
+```py
+from torchvision.transforms import Compose, ToTensor, Resize, Normalize, ToPILImage
+
+pre_proccess = Compose([
+    ToTensor(),
+    Normalize([.485, .456, .406], [.229, .224, .225])
+])
+
+demo_img_path = "/home/data/supervisely-persons/images/ache-adult-depression-expression-41253.png"
+img = Image.open(demo_img_path)
+# Resize the image and display
+img = Resize(size=(480, 320))(img)
+display(img)
+
+# Run pre-proccess - transforms to tensor and apply normalizations.
+img_inp = pre_proccess(img).unsqueeze(0).cuda()
+
+# Run inference
+mask = model(img_inp)
+
+# Run post-proccess - apply sigmoid to output probabilities, then apply hard
+# threshold of 0.5 for binary mask prediction. 
+mask = torch.sigmoid(mask).gt(0.5).squeeze()
+mask = ToPILImage()(mask.float())
+display(mask)
+```
+![segmentation_prediction.png](segmentation_prediction.png)
+
+
+## Going further
+### How to launch on multiple GPUs (DDP) ?
+To run the Training using Distributed Data Parallel (DDP), all you need to do is to call a magic function `setup_device` before instantiating the Trainer.
```

Comment:
Instead, add references to other pages (DDP, using recipes...)

Reply:
Changed in d1fb7168b1580600f5594a42819fa8246042c207
It's just a placeholder for now, I have a PR open to add explanation on how to use DDP (https://github.com/Deci-AI/super-gradients/pull/662)